bigtable_automation/gcf/private.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package private runs a GCF function that triggers in 2 scenarios:
+//
+// 1) completion of prophet-flume job in borg. On triggering it sets up new
+//    cloud BT table, scales up BT cluster (if needed) and starts a dataflow job.
+// 2) completion of BT cache ingestion dataflow job. It scales BT cluster down
+//    (if needed).
+// The trigger function BTImportController requires a set of environment
```

Comment:
fix name

Reply:
Removed these since the package comment is in common.go